app/controllers/application_controller.rb

```diff
@@ -183,4 +184,10 @@ def timestamp_with_usec
   def debug_msg(msg)
     "[#{timestamp_with_usec}] #{msg}"
   end
+
+  private
+  def set_last_seen_at
+    current_user.update_attribute(:last_seen_at, Time.now)
```

Comment:
Any reason you are using `update_attribute`?  It is ok but it skips callbacks, validations etc.
A simple `update` works fine in rails 4 (or you can use `update_column` if you want to skip validations etc.